src/applications/personalization/profile-2/components/MilitaryInformation.jsx

```diff
@@ -97,6 +97,7 @@ const MilitaryInformationContent = ({ militaryInformation }) => {
         dataTransformer={transformServiceHistoryEntryIntoTableRow}
         title="Period of service"
         fieldName="serviceHistory"
+        list
```

Comment:
this is a new optional prop on the ProfileInfoTable that adds additional markup to its rendered output (more details in this PR)